src/NuGet/Microsoft.CodeAnalysis.CSharp.Features.nuspec

```diff
@@ -4,6 +4,8 @@
     <id>Microsoft.CodeAnalysis.CSharp.Features</id>
     <description>
```

Comment:
So I think any package that doesn't have a summary tag now needs to get one: otherwise you'll see our URL being randomly plastered in the NuGet package selection UI and such. For example:

![image](https://cloud.githubusercontent.com/assets/201340/26259740/3abb5632-3c7f-11e7-8e07-1a2e657ec8b3.png)

The one-line thing in the left is the summary, the right longer thing is the description. The URL is definitely not something useful for a one-line description of the package.